WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/CardPresentPaymentsOnboardingUseCase.swift

```diff
@@ -337,7 +337,7 @@ private extension CardPresentPaymentsOnboardingUseCase {
                 return .codPaymentGatewayNotSetUp(plugin: plugin)
             }
         }
-        guard !isInUndefinedState(account: account) else {
+        guard canCollectCardPresentPayments(account: account) else {
```

Comment:
```suggestion
        guard accountStatusAllowedForCardPresentPayments(account: account) else {
```
A bit of a nit-pick but it seems more specific to what we're checking

Reply:
Indeed, that's more specific, thanks for the comment, changed in 15b918d9f9e786d44943060f23e22b7ed5ff4658